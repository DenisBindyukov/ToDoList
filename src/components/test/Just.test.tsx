import React from  "react";
import {shallow} from "enzyme";
import {Label} from  './Label'
import {configure} from 'enzyme';
import renderer from 'react-test-renderer'
const Adapter = require('enzyme-adapter-react-16')

configure({ adapter: new Adapter() });

describe('Label', () => {
    it('should have default render', () => {
        const wrapper = renderer.create(<Label/>).toJSON()
        expect(wrapper).toMatchSnapshot();
    })

    it('should have render with props', () => {
        const wrapper = shallow(<label className={'mock-label'} htmlFor={'unknown-input'}/>);
        expect(wrapper).toMatchSnapshot()
    })
})