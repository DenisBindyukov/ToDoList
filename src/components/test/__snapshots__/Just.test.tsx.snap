// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should have render with props 1`] = `ShallowWrapper {}`;
